src/server/command/debugCommand.ts

```diff
@@ -0,0 +1,101 @@
+import { AnyChannel, Client, Message } from 'discord.js';
+
+async function getCommand(client: Client, message: Message, prefix: string) {
+  if (message.author.bot || !message.guild) return;
+
+  const command = `${prefix || '!'}debug`;
+  const str = message.content;
+  const match = str.startsWith(command);
+  if (!match) return;
+
+  const commandArgs = str.slice(prefix?.length).trim().split(' ');
+  const argsMessageId = commandArgs[1];
+  if (!argsMessageId) {
+    await message.reply({
+      content: `**メッセージIDが指定されていません。**\n Usage: \`${
+        prefix || '!'
```

Comment:
`prefix` はすでに `string` 型なので, `?` や `||` による防護は不要です
```suggestion
  const command = `${prefix}debug`;
  const str = message.content;
  const match = str.startsWith(command);
  if (!match) return;

  const commandArgs = str.slice(prefix.length).trim().split(' ');
  const argsMessageId = commandArgs[1];
  if (!argsMessageId) {
    await message.reply({
      content: `**メッセージIDが指定されていません。**\n Usage: \`${
        prefix
```